target/start-mailserver.sh

```diff
@@ -1483,14 +1483,16 @@ function _setup_postfix_dhparam
     if [[ ! -f ${DHPARAMS_FILE} ]]
     then
       _notify 'inf' "Use ffdhe4096 for dhparams (postfix)"
-      rm -f /etc/postfix/dhparams.pem && cp /etc/postfix/shared/ffdhe4096.pem /etc/postfix/dhparams.pem
+      rm -f /etc/postfix/dhparams.pem &>/dev/null
```

Comment:
Are these rm statements necessary, when the file is recreated in the next line?

Reply:
Actually not, you're right. I just confirmed locally that `cp`-ing will just overwrite this file. I've pushed a new commit, thanks!